src/lib/Telemetry/telemetry.cpp

```diff
@@ -62,6 +67,20 @@ void Telemetry::CheckCrsfBatterySensorDetected()
     }
 }
 
+void Telemetry::SetCrsfBaroSensorDetected(bool on)
+{
+        crsfBaroSensorDetected = on;
+}
+
+void Telemetry::CheckCrsfBaroSensorDetected()
+{
+    if (CRSFinBuffer[CRSF_TELEMETRY_TYPE_INDEX] == CRSF_FRAMETYPE_BARO_ALTITUDE ||
+        CRSFinBuffer[CRSF_TELEMETRY_TYPE_INDEX] == CRSF_FRAMETYPE_VARIO)
+    {
+        crsfBaroSensorDetected = true;
```

Comment:
Call SetCrsfBaroSensorDetected() so crsfBaroSensorDetected is only set in a single location.

And the same comment for crsfBatterySensorDetected.

Reply:
Done